### PR TITLE
fixed introduceMe function: FAILED

### PR DESCRIPTION
**Pull request by Hanifix**
fixed introduceMe function: FAILED
i am strugling with this part

### CODE REVIEW

**Comment by Hanifix** [comment]
This is my first update for the 2-logic

File: week-1/Homework/mandatory/2-logic-error.js [hunk 1]
```diff
@@ -1,31 +1,31 @@
 // The syntax for this function is valid but it has an error, find it and fix it.
 
 function trimWord(word) {
-  return wordtrim();
+    return word;
```
Comment (amandaeareias) [inline week-1/Homework/mandatory/2-logic-error.js]: In this function you're supposed to find a method that trims the string (removes whitespace from both ends of it). For example:
`   hanif    ` would became `hanif`

You don't need to do it yourself, JavaScript has built in methods that you can use on strings. You can try looking for it on google and implement it here

File: week-1/Homework/mandatory/2-logic-error.js [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
 }
 
 function getWordLength(word) {
-  return "word".length()
+    return word;
```
Comment (amandaeareias) [inline week-1/Homework/mandatory/2-logic-error.js]: In this function we want to get the word's **length** (quantity of characters in it). For example: 
if **word** is `hanif` the result would be `6`.

Same as above: You don't need to do it yourself, JavaScript has built in methods that you can use on strings. You can try looking for it on google and implement it here

**Comment by Hanifix** [comment]
This are some modification i have done

**Pull request closed without merging**